src/drive/hooks/useRedirectLink.spec.jsx

```diff
@@ -88,44 +99,78 @@ describe('useRedirectLink', () => {
         }
       ]
     })
-
+    useSearchParams.mockReturnValue([
+      new URLSearchParams('?redirectLink=drive%23%2Ffolder%2Fid123')
+    ])
     let render
     await act(async () => {
-      render = renderHook(() => useRedirectLink())
+      render = renderHook(() => useRedirectLink({ isPublic: true }))
     })
 
-    expect(render.result.current.redirectWebLink).toBe(
+    render.result.current.redirectBack()
+
+    expect(mockNavigate).toHaveBeenCalledTimes(0)
+    expect(mockAssign).toHaveBeenCalledWith(
```

Comment:
@Crash-- do you know how to spy a `window.location` with jest? I tried several technique but I can't found a way to make it work